tests/test_helper/bats/upgrade.bats

```diff
@@ -19,5 +19,21 @@ setup() {
     echo "# Upgrading MicroOVN from revision $MICROOVN_SNAP_REV" >&3
     install_microovn "$MICROOVN_SNAP_PATH" $TEST_CONTAINERS
     perform_manual_upgrade_steps $TEST_CONTAINERS
-    # TODO: include more cluster and TLS tests once we settle on how to include them without copy-pasting
+    local cluster_test_filename="${ABS_TOP_TEST_DIRNAME=}test_helper/bats/post_upgrade_cluster.bats"
+    local tls_test_filename="${ABS_TOP_TEST_DIRNAME=}test_helper/bats/post_upgrade_tls.bats"
+
+    # Note that the outer bats runner will perform validation on the
+    # number of tests ran based on TAP output, so it is important that
+    # the inner bats runner uses a different format for its output.
+    run bats -F junit $cluster_test_filename
```

Comment:
nit: The cluster test is IPv6-aware. Should we run both the IPv4 and IPv6 version of it?

The scaleup test accomplishes this with an extra symlink and this in the code:
https://github.com/canonical/microovn/blob/2a0e6b556a02074774dcc5fc50f835a8f2165b50/tests/scaleup_cluster.bats#L60